### ISSUE TITLE
Download counts are not the same as in NPM. Are they downloads per week?

### ISSUE THREAD

**Issue by vibl** [uidotdev/npm-trends#181]
Download counts are not the same as in NPM. Are they download per week?

Example:

https://www.npmjs.com/package/react-aria : ~41k downloads per week
https://npmtrends.com/react-aria:  ~150k downloads per ?

Also, the last number is always much lower than the rest, which gives the impression that all packages downloads have been plummeting recently (which is probably not the case given that I'm writing this on Thursday September 21st).

**Comment by abizek** [comment]
Yes, these are downloads per week. According to the code, npmtrends discards data from the current week.

https://github.com/uidotdev/npm-trends/blob/a5b2fdb3d74d6842f319ff85f294d489214a8953/pages/%5B%5B...packets%5D%5D.tsx#L64
